registration-processor/registration-processor-registration-status-service-impl/src/test/java/io/mosip/registration/processor/status/service/RegistrationStatusServiceTest.java

```diff
@@ -203,6 +253,21 @@ public void getByIdsFailureTest() {
 		registrationStatusService.getByIds(registrationIds);
 
 	}
+	
+	@Test(expected = TablenotAccessibleException.class)
+	public void getExternalStatusByIdsFailureTest() {
+		RegistrationExternalStatusSubRequestDto registrationId = new RegistrationExternalStatusSubRequestDto();
+		registrationId.setRegistrationId("1001");
+		List<RegistrationExternalStatusSubRequestDto> registrationIds = new ArrayList<>();
+		registrationIds.add(registrationId);
+
+		DataAccessLayerException exp = new DataAccessLayerException(HibernateErrorCode.ERR_DATABASE.getErrorCode(),
+				"errorMessage", new Exception());
+		Mockito.when(registrationStatusDao.getByIds(any())).thenThrow(exp);
+
+		registrationStatusService.getExternalStatusByIds(registrationIds);
+
+	}
 
 	@Test
```

Comment:
We have move the full external status computation in the registration status service, can you add 3 to 4 test case on that external status logic for main process and additional info process

Reply:
wrttien test cases.